README.md

```diff
@@ -95,6 +95,25 @@ envsubst < helm/admin/secrets.yaml | kubectl apply -n "$NAMESPACE" -f -
 envsubst < helm/admin/postgres-secrets.yaml | kubectl apply -n "$NAMESPACE" -f -
 ```
 
+5. Create OGDC database.
+
+Install CNPG:
+
+```
+helm repo add cnpg https://cloudnative-pg.github.io/charts
+helm upgrade --install cnpg \
+  --namespace cnpg-system \
+  --create-namespace \
```

Comment:
We don't generally let charts create namespaces on our cluster. We create them ahead of time and assign RBAC controls to them, then let authorized service accounts install into that namespace.